DesignSafe-CI/DS-User-Guide
# enhance: GH-44 math via tacc-docs

## Overview

Move MathJax source of truth to https://github.com/TACC/TACC-Docs.

> [!WARNING]
> - Makes support implicit, cuz it works always without clear reason.
> - Makes support hidden, cuz config is in private[^1] TACC-Docs.

[^1]: It should become public next year.

## Related

- standardizes #44

## Testing

0. Open http://0.0.0.0:8000/user-guide/usecases/padgett/usecase_JN_viz/#Title2
1. Verify the math (see UI) is **still** rendered as math, not "$…$".

## UI

<img width="960" alt="Screenshot 2024-12-20 at 17 26 36" src="https://github.com/user-attachments/assets/8c1377a4-a721-4ddb-ba5b-403b279cc265" />

## Notes

This is a **middle-term** solution is [to update config in TACC/TACC-Docs](https://github.com/DesignSafe-CI/DS-User-Guide/issues/44#issuecomment-2173971202). The [long-term](https://github.com/DesignSafe-CI/DS-User-Guide/issues/44#issuecomment-2173971202) solution is **not** use Docker to get TACC skin (#42), so that DS-User-Guide can update is MkDocs config without worrying about skin.
